### ISSUE TITLE
Match stats contrast issue

### ISSUE THREAD

**Issue by cemms1** [guardian/dotcom-rendering#8363]
Feedback about Match Stats on mobile web (but same issue occurs on desktop view too):

>The colour contrast isn’t there in the same way it is on app (iOS), which makes it very difficult to understand. 
>
>Plus on mobile web, you need to scroll past the stats (which are tabbed on app) to get to the blog so it’s not a great reader experience when it looks like this. 

| Mobile web | iOS app |
| ---------- | ------- |
| ![image](https://github.com/guardian/dotcom-rendering/assets/43961396/d2688801-e1df-48b0-81a4-02f063591863) | ![image](https://github.com/guardian/dotcom-rendering/assets/43961396/27bcad84-100b-4d54-8c88-79df0d137f38) |

Example article:
https://www.theguardian.com/football/2023/jul/22/england-haiti-womens-world-cup-match-report

Seems to happen when two team colours are very similar, but iOS handles this better

**Comment by VDuczekW** [comment]
Let's have a look and see how much effort this piece of work to tweak - or is it worth doing as a bigger change - check with design @cemms1

**Comment by VDuczekW** [comment]
Anna talk to design before going forward

**Comment by VDuczekW** [comment]
waiting on Alex B - response

**Comment by VDuczekW** [comment]
it has been passed on to design - new ticket will be created for wider work - but PA as have done an update and better contrast applied

**Comment by abeddow91** [comment]
Updated team colours requested from PA made a significant improvement 

![Image](https://github.com/guardian/dotcom-rendering/assets/20416599/857400c9-6542-41c6-bb66-1bc60e987844)
![Image](https://github.com/guardian/dotcom-rendering/assets/20416599/72a11eac-7011-4822-b03a-af823d50cfaa)

**Comment by abeddow91** [comment]
https://github.com/orgs/guardian/projects/88/views/2?pane=issue&itemId=35332044 <- new ticket available here
